### ISSUE TITLE
Blocks: Full saved cards support

### ISSUE THREAD

**Issue by RadoslavGeorgiev** [Automattic/woocommerce-payments#950]
- [ ] Modify the `WCPay\Payment_Information` class to work with the right variables in order to allow the usage of existing saved cards. Currently an `Invalid payment method. Please input a new card number.` exception gets thrown.
- [ ] Display the checkbox to save cards in the checkout block and connect the logic behind it.
- [ ] Add E2E tests
